server.js

```diff
@@ -27,22 +27,32 @@ var polling = AsyncPolling(end => {
         }
     };
 
-    request(options, (error, response, body) => { 
-        const mediaItems = JSON.parse(body).data.mediaItems;
+    request(options, (error, response, body) => {
+        
+        if (error || response.statusCode !== 200) [
```

Comment:
Please JSlint or ESlint your code in the future. Partly my fault for not running it before merging it.